package.json

```diff
@@ -33,11 +33,11 @@
     "test": "echo 'placeholder. no tests implemented'"
   },
   "devDependencies": {
-    "@folio/eslint-config-stripes": "^3.2.1",
+    "@folio/eslint-config-stripes": "^5.2.0",
     "@folio/stripes": "^3.0.0",
```

Comment:
hmm. hmmmmmm

Reply:
yes, I've just noticed there is PR with all updates https://github.com/folio-org/ui-servicepoints/pull/88

Reply:
Noice!!